packages/embark/src/lib/modules/ethereum-blockchain-client/index.js

```diff
@@ -54,23 +58,19 @@ class EthereumBlockchainClient {
 
   async deployer(contract, done) {
     const web3 = await this.web3;
```

Comment:
Why does this need to be an await?

Reply:
It's a special `get web3` thingy. It returns a promise that resolves with the existing web3 object or a new one if it didn't exist. See line 39

Reply:
Ah gotcha!